router/userRouter.js

```diff
@@ -11,10 +11,56 @@ import registerRules from "../validation/registerRules.js";
 
 const router = express.Router();
 
-router.get("/", getUsers);
-router.post("/add", validate(registerRules), addUser);
-router.post("/login", loginUser);
-router.put("/update", updateUser);
-router.delete("/delete", deleteUser);
+// router.get("/", getUsers);
+// router.post("/add", validate(registerRules), addUser);
+// router.post("/login", loginUser);
+// router.put("/update", updateUser);
+// router.delete("/delete", deleteUser);
+
+
+// ------ #issue5-endpoints work on 16/11/2021
+// just experimenting w/ endpoints --> this is not functional and will not run
+
+// routes to browse events without being a registered and logged in user
+router.get("/", getEventsForToday); // landing page
+router.get(`/events/${eventId}`, getEvent(eventId)); // click on one event
```

Comment:
If this is something like `/user/events/52` how would it be different from what `/event/52` would be? 

If the event id is a route parameter like `/users/events/52` this would be `/events/:eventId` and where you have `getEvent(eventId)` there would actually just be a function reference `getEvent`, just like the `getEventsForToday` above.  Most likely actually these functions would be best if they were like `userController.getEvent` and imported like `import * as userController from '../controllers/userController.js` or something like that :) 

If the function that will handle this is called getEvent,

Reply:
I was wondering, if ${eventId} and getEvent(eventId) was even gonna work, as in the userController getEvent function, you could only use error, req, res and next as parameters, but nothing else, right? That was a thought i had at some point during the night...
My question was, if i can use :eventId here (in userRouter.js), if i only use req.params.eventId in userController.js. But that probably works, if you can also use req.query there as i learned in your next comment.